projects/core/src/lib/layers/symbology/raster-symbology-editor/raster-symbology-editor.component.html

```diff
@@ -1,129 +1,153 @@
-                    ></geoengine-vega-viewer>
-                </ng-container>
-                <mat-progress-bar mode="indeterminate" *ngIf="histogramLoading | async"></mat-progress-bar>
-            </div>
-            <div>
-                <mat-slide-toggle [checked]="histogramAutoReload" (change)="histogramAutoReload = $event.checked">
-                    Sync map and histogram
-                </mat-slide-toggle>
-            </div>
-            <mat-divider></mat-divider>
+                    <geoengine-color-map-selector
+                        (breakpointsChange)="updateBreakpoints($event)"
+                        (changesToForm)="getNotified()"
+                        [minValue]="layerMinValue"
+                        [maxValue]="layerMaxValue"
+                        [scale]="scale"
+                    >
+                    </geoengine-color-map-selector>
+                </ng-template>
+            </mat-expansion-panel>
 
-            <geoengine-color-map-selector
-                (breakpointsChange)="updateBreakpoints($event)"
-                [minValue]="layerMinValue"
-                [maxValue]="layerMaxValue"
-                [scale]="scale"
-            >
-            </geoengine-color-map-selector>
-        </ng-template>
-    </mat-expansion-panel>
+            <!-- TODO: implement palette editor -->
+            <mat-expansion-panel expanded="false" *ngIf="symbology.colorizer.isDiscrete()">
+                <mat-expansion-panel-header>
+                    <mat-panel-title fxLayoutAlign="space-between center" fxLayoutGap="0.5rem">
+                        <mat-icon>color_lens</mat-icon>
+                        <span fxFlex>Color Table</span>
+                    </mat-panel-title>
+                </mat-expansion-panel-header>
+                <ng-template matExpansionPanelContent>
+                    <!-- <geoengine-colorizer-editor [ngModel]="symbology.colorizer" (ngModelChange)="updateColorizer($event)"></geoengine-colorizer-editor> -->
+                </ng-template>
+            </mat-expansion-panel>
+        </mat-accordion>
+    </div>
 
-    <!-- TODO: implement palette editor -->
-    <mat-expansion-panel expanded="false" *ngIf="symbology.colorizer.isDiscrete()">
-        <mat-expansion-panel-header>
-            <mat-panel-title fxLayoutAlign="space-between center" fxLayoutGap="0.5rem">
-                <mat-icon>color_lens</mat-icon>
-                <span fxFlex>Color Table</span>
-            </mat-panel-title>
-        </mat-expansion-panel-header>
-        <ng-template matExpansionPanelContent>
-            <!-- <geoengine-colorizer-editor [ngModel]="symbology.colorizer" (ngModelChange)="updateColorizer($event)"></geoengine-colorizer-editor> -->
-        </ng-template>
-    </mat-expansion-panel>
-</mat-accordion>
+    <div class="actions">
+        <span class="hint" *ngIf="unappliedChanges === true">Changes not reflected in map view!</span>
```

Comment:
```suggestion
        <span class="hint" *ngIf="unappliedChanges === true">Changes are not yet reflected in map view!</span>
```

Reply:
But actually it is quite clear because you see the Apply changes button that you can click on all the time.

Maybe you can remove the message.

Moreover, maybe we should add a reset button there instead (not in primary color). What do you think?

Reply:
I added a reset button that reloads the component. It's using the "accent" color for now.
Also removed the message about unapplied changes. f2c0f44af28f79d4d5bb91812260706d2c463892
![image](https://user-images.githubusercontent.com/57327151/203059689-d36b8f67-b545-487f-b10e-fc7f333834f3.png)

Reply:
Please switch the places (left to right) – it sees more natural, doesn't it?

Moreover, clicking reset (good idea by the way) should not recompute the histogram.

Reply:
Switched the button positions, and the histogram isn't calculated on a reset df5f7fdfc5f3388cdcb12b1a05092ac86301c885